charts/pgadmin4/templates/secrets.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if not .Values.existingSecret }}
```

Comment:
I don't see this value coming back within values.yaml with an example. Perhaps it would be best if we added it commented to make people aware it is an option?